txnprovider/txpool/pool_txn_parser.go

```diff
@@ -690,13 +690,16 @@ func parseTransactionBodyAA(payload []byte, p int, slot *TxnSlot) (int, error) {
 	if err != nil {
 		return 0, err
 	}
-	slot.SenderAddress = &address
+	slot.SenderAddress = &common.Address{}
+	copy((slot.SenderAddress)[:], address[:])
+	copy(sender, address[:])
 
 	address, p, err = getAddress(payload, p, "deployerAddress")
 	if err != nil {
 		return 0, err
 	}
-	slot.Deployer = &address
+	slot.Deployer = &common.Address{}
```

Comment:
@shohamc1 
This appears to be a bug with assigning the same pointer to three different fields, therefore overriding the actual data.